tutorial/debezium-with-apicurio/Dockerfile

```diff
@@ -0,0 +1,9 @@
+ARG DEBEZIUM_VERSION
+FROM debezium/connect:$DEBEZIUM_VERSION
+ENV KAFKA_CONNECT_DEBEZIUM_DIR=$KAFKA_CONNECT_PLUGINS_DIR/debezium-connector-mysql
+
+COPY *.tar.gz $KAFKA_CONNECT_DEBEZIUM_DIR/apicurio.tar.gz
+
+RUN  cd $KAFKA_CONNECT_DEBEZIUM_DIR &&\
+    tar xvf apicurio.tar.gz &&\
+    rm apicurio.tar.gz
```

Comment:
Can you add a note to the README on this binary, its purpose and future plan to remove to upstream artifact?

Reply:
@gunnarmorling Would not it be better to comment directly in Dockerfile?

Reply:
@gunnarmorling Commetn added to the Dockerfile